docs/book/codegen.md

```diff
@@ -75,6 +75,9 @@ following keys (unknown keys are ignored):
   this value is not provided, you will need to set it with the generator's
   `setOutputDirectory()` method before calling `generate()`.
 
+- `logger`: By default `Psr\Log\NullLogger` is defined. See the
```

Comment:
I'd say "is used" instead of "is defined".

Also, I would be more descriptive here - `logger` must be resolvable in container and must be an instance of `Psr\Log\LoggerInterface`.

Reply:
Agree. 

@waahhhh could you please address this?